internal/ignition/ignition.go

```diff
@@ -278,25 +320,38 @@ func (g *installerGenerator) createLsoManifests() error {
 	}
 	return nil
 }
-func (g *installerGenerator) generateLsoManifests(ctx context.Context, installerPath string, envVars []string) error {
-	if g.checkLsoEnabled() {
+
+func (g *installerGenerator) generateOperatorsManifests(ctx context.Context, installerPath string, envVars []string, ocsValidatorConfig *ocs.Config) error {
+	lsoEnabled := g.checkOcsEnabled()
+	ocsEnabled := g.checkLsoEnabled()
```

Comment:
Seems like an assignment mismatch?
Also, if OCS is enabled, LSO needs to be enabled too.

Reply:
fixed